src/org/ods/component/ScanWithSnykStage.groovy

```diff
@@ -56,7 +56,8 @@ class ScanWithSnykStage extends Stage {
             "buildFile=${config.buildFile}, " +
             "failOnVulnerabilities=${config.failOnVulnerabilities}, " +
             "severityThreshold=${config.severityThreshold}, " +
-            "additionalFlags=${config.additionalFlags ? config.additionalFlags.toListString() : ''}."
+            "additionalFlags=${config.additionalFlags ? config.additionalFlags.toListString() : ''}." +
+            "${logger.debugMode ? '-d' : ''}"
```

Comment:
Personally I think we don't need the extra info about the debug mode here ...

Reply:
I think it's worthwhile, especially when calling external jars / tools